lib/api/cln/cln_client.dart

```diff
@@ -55,7 +55,16 @@ class CLNApi extends AppApi {
             jsonResponse as Map<String, dynamic>,
             snackCase: !mode.withCamelCase()));
     var listFunds = await this.listFunds();
+    var invoicesList = await listInvoices(status: "paid");
+
+    /// Adding the listfunds amounts
     appInfo.totOffChainMsat = listFunds?.totOffChainMsat ?? 0;
+
+    /// Adding the paid invoices to the list
+    for (AppInvoice invoice in invoicesList.invoice) {
+      int parse = int.parse(invoice.amount);
```

Comment:
please note from cln docs

```
If status is "paid":

pay_index (u64): Unique incrementing index for this payment
amount_received_msat (msat): the amount actually received (could be slightly greater than amount_msat, since clients may overpay)
paid_at (u64): UNIX timestamp of when it was paid
payment_preimage (secret): proof of payment
```